### DISCUSSION TITLE
Management webpage not populating with 3.9.8

### DISCUSSION THREAD

**Discussion by mrodriguez-aviles** [rabbitmq/rabbitmq-server#3612] [Other] [answered]
I wanted to see if anybody else is having a problem with the management webpage (http://localhost:15672/#/ )  in which after you log in it shows basically an empty screen.   I ran into this issue when upgrading from 3.8 to 3.9.8 or with a fresh server install using 3.9.8.

Example of steps:
choco install rabbitmq

Start RabbitMQ Node
rabbitmqctl start_app

Enable RabbitMQ Management UI: rabbitmq-plugins enable rabbitmq_management

Using a browser on the server, access the http://localhost:15672/ 
The default administrator username and password are guest and guest.
![Screenshot 2021-10-26 141417](https://user-images.githubusercontent.com/58010854/138946393-f6f48842-9ede-4bc2-859c-ada4568b760a.png)
![rabbit_fine](https://user-images.githubusercontent.com/58010854/138946400-701d2942-2bb4-4782-a94b-6924201decac.png)

**Comment by mkuratczyk** [comment, marked as answer]
Such issues are usually due to browser caching. Please clear the cache, hard refresh the page or try a browser you didn't use before.

**Reply by mrodriguez-aviles** [reply]
Thank you.  I ended up installing Google Chrome on that server with the fresh install and now it shows the node info as desired.
